internal/ioutil/io.go

```diff
@@ -46,13 +46,20 @@ func CopyBuffer(dst io.Writer, src io.Reader, buf []byte, desc ocispec.Descripto
 
 // nopCloserType is the type of `io.NopCloser()`.
 var nopCloserType = reflect.TypeOf(io.NopCloser(nil))
+var nopCloserWriterToType = reflect.TypeOf(io.NopCloser(struct {
```

Comment:
request documentation for `nopCloserWriterToType`

Reply:
added this: 
// nopCloserWriterToType is the type of `io.nopCloserWriterTo`